docs/src/wiki/expanded_pathogens.rst

```diff
@@ -0,0 +1,77 @@
+Running Freyja on other pathogens
+-------------------------------------------------------------------------------
+
+This guide provides instructions for analyzing non-SARS-CoV-2 pathogens such as
+influenza or MPox using Freyja. The process is similar to SARS-CoV-2 analysis,
+but with some key differences.
+
+Data Availability
+^^^^^^^^^^^^^^^^^
+
+Data for various pathogens can be found in the following repository:
+`Freyja Barcodes <https://github.com/gp201/Freyja-barcodes>`_
```

Comment:
Might be worth adding something about flu barcodes- maybe we set up a private barcode repo just for those? (Since Freyja-process-barcodes will need to stay private, we could just give access to that?)